src/main/java/edu/wpi/first/nativeutils/vendordeps/VendorDepTask.java

```diff
@@ -41,6 +45,22 @@ public void install() throws IOException {
             getLogger().info("Remotely fetching " + filename);
             downloadRemote(dest);
         }
+
+        var destString = dest.toString();
+        String newFilename;
+        try (BufferedReader reader = Files.newBufferedReader(dest)) {
+            newFilename = new GsonBuilder().create().fromJson(reader, JsonDependency.class).fileName;
```

Comment:
How will this behave with a malformed vendor JSON? eg one missing the fileName attribute

Reply:
Crash. I'll add a try catch

Reply:
I lie. If it doesn't have fileName file name will be null. Crashing would be caused by Json syntax issues or IO issues